tests/integration/exec.bats

```diff
@@ -57,3 +57,42 @@ function teardown() {
   [[ ${lines[1]} == *"."* ]]
   [[ ${lines[2]} == *".."* ]]
 }
+
+@test "runc exec ls -la with --cwd" {
+  # run busybox detached
+  runc run -d --console /dev/pts/ptmx test_busybox
+  [ "$status" -eq 0 ]
+
+  wait_for_container 15 1 test_busybox
+
+  runc exec --cwd /bin test_busybox pwd
+  [ "$status" -eq 0 ]
+
+  [[ ${output} == *"/bin"* ]]
```

Comment:
thanks for your review.

but `[[ $(echo "${output}" | tr -d ' \n') == "/bin" ]]` and `[[ ${lines[0]} == "/bin" ]]` do not works. the same as `[[  ${output} == "/bin" ]]`.

and I write a simple test `bats` scripts, it test ok

```
#!/usr/bin/env bats

@test "pwd test" {
   run runc exec --cwd /bin test_busybox pwd
   [[ ${output} == "/bin" ]]
}
```
So I don't know why they all do not work in `tests/integration/*.bats`?